apps/intellij/src/main/kotlin/dev/nx/console/generate/run_generator/RunGeneratorManager.kt

```diff
@@ -105,14 +107,33 @@ class RunGeneratorManager(val project: Project) {
         }
     }
 
-    private fun setProcessHandler(processHandler: KillableColoredProcessHandler) {
+    private fun setProcessHandler(
+        processHandler: KillableColoredProcessHandler,
+        refreshFileSystem: Boolean
+    ) {
         processHandler.addProcessListener(
             object : ProcessListener {
                 override fun processTerminated(event: ProcessEvent) {
                     runningProcessHandler = null
+                    if (refreshFileSystem) {
+                        VfsUtil.markDirtyAndRefresh(
+                            true,
+                            true,
+                            true,
+                            VfsUtil.findFileByIoFile(File(project.nxBasePath), true)
+                        )
+                    }
                 }
             }
         )
         runningProcessHandler = processHandler
     }
 }
+
+private fun List<String>.isDryRun(): Boolean {
+    return this.contains("--dry-run") || this.contains("--dryRun")
+}
+
+private fun String.isDryRun(): Boolean {
+    return this.contains("--dry-run") || this.contains("--dryRun")
+}
```

Comment:
I'm a little iffy on this. But we'll see in the future

Reply:
you mean it shouldn't be an extension function?